### ISSUE TITLE
Site produces some minor html validation errors

### ISSUE THREAD

**Issue by geraldbaeck** [mkdocs/mkdocs#1716]
https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.mkdocs.org%2F

- the id "exampleModalLabel" is used within the search-modal and the keyboard modal
- "aria-labelledby" is not used correctly also within the search-modal and the keyboard modal

**Comment by waylan** [comment]
Thanks for the report.

It appears that the `id="exampleModalLabel"` was simply copied from a bootstrap example unaltered for both the search modal and the keyboard modal. At least we should give each a unique (and relevant) name.

The purpose of `aria-labelledby` is described clearly in [Mozilla's docs][1]. Specifically, it should be a space separated list of related ids. The examples in the [Bootstrap docs][2], include a single id which points to the id of the modal's title. That is, it should point to what is currently `exampleModalLabel`. Therefore, each instance needs to be updated to point at the proper id once that id is renamed with a unique name.

Of course, PRs are welcome.

[1]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute
[2]: https://getbootstrap.com/docs/4.1/components/modal/#live-demo
